src/course/syllabus/developer/project-1-server/project.md

```diff
@@ -1,26 +1,36 @@
-# AI Learning Tool
+# The Amazin' Quizzer App
 
-Learners will develop an AI-driven web application that dynamically generates quizzes for users in a specified subject area. These quizzes will have interfaces both for quiz creation and quiz taking.
+## Intro
+
+Learners will develop an web application that dynamically generates quizzes. These app should have interfaces both for quiz creation and quiz taking. You can use a AI tool to generate questions or use the ones provided and should use a JSON object in a separate file as a mock database. 
+
+## Project
+
+Your project for this week will be to develop a RESTful api as the backend of your application.
+
+## Spike
+
+Before you start writing features you need to design the schema for your data and RESTful API. Think about what different data your app needs to store and how you will access that data using your endpoints. Record your schema and endpoints in your `README.md` to make it easy for other developers to quickly get started using your API. 
 
 ### User Stories
 
-As a **learner**, I want to:
+As a **quizzer**, I want to:
 
 - Select a subject area for my quiz.
 - Begin a quiz session with randomly generated questions.
 - View a summary of my quiz results, including areas of strength and weakness.
 - Add, edit, and delete quiz questions and answers.
-- Generate reports on quiz performance
 
 ### Acceptance Criteria
 
-**The application should:**
+**The API should:**
+
+- [ ] Dynamically generate quizzes with adaptive difficulty.
+- [ ] Filter quiz questions by category.
+- [ ] Return any number of questions by the above criteria.
+- [ ] Add or delete questions from the 'database'.
```

Comment:
There's nothing wrong with this but we could also use the common naming: **CRUD**
- Create
- Read
- Update
- Delete

That's pretty common language, particularly in the .Net world, so it's nice to have them learn it early on